spec/requests/authentication/standard_authentication_spec.rb

```diff
@@ -20,7 +20,7 @@
     end
   end
 
-  xit 'does the tests', :aggregate_failures, :skip_mvi, :skip_emis do
+  xit 'does the tests', :aggregate_failures, :skip_mvi do
```

Comment:
🗣️ I wonder why this spec got skipped. [This PR](https://github.com/department-of-veterans-affairs/vets-api/pull/7199) introduced the change, but I don't see any comments as to why. Definitely not related to your current effort, so don't worry about it; I'll circle back around to dig deeper later.

Reply:
🤔 🤔 I wonder as well. Good luck on that wild goose chase 😆 I saw it was commit https://github.com/department-of-veterans-affairs/vets-api/pull/7199/commits/a8ff85f180a8dca507551ef558849d547b6f508b where it was skipped. 

Thanks for the review!